docs/source/examples/performance-training.rst

```diff
@@ -0,0 +1,402 @@
+.. _high-performance-training:
+
+High Performance Training
+=========================
+
+This guide covers best practices and examples for achieving high performance distributed training using SkyPilot.
+
+Distributed training basics
+----------------------------
+
+SkyPilot supports various distributed training frameworks and approaches:
+
+- `PyTorch Distributed Data Parallel (DDP) <https://github.com/skypilot-org/skypilot/tree/master/examples/spot/resnet_ddp>`_
+- `Horovod <https://github.com/skypilot-org/skypilot/tree/master/examples/horovod>`_
+- `DeepSpeed <https://github.com/skypilot-org/skypilot/tree/master/examples/deepspeed>`_
+- `Ray Train <https://github.com/skypilot-org/skypilot/tree/master/examples/ray_train>`_
+- `TensorFlow Distribution Strategies <https://github.com/skypilot-org/skypilot/tree/master/examples/tensorflow>`_
+
+The choice of framework depends on your specific needs, but all can be easily configured through SkyPilot's YAML specification.
+
+Best practices
+--------------
+
+High performance instances
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Choose high performance instances for optimal training performance. SkyPilot allows you to specify instance types with powerful GPUs and high-bandwidth networking:
+
+- Use the latest GPU accelerators (A100, H100, etc.) for faster training
+- Select instances with NVLink for improved multi-GPU communication
```

Comment:
As a reader I now wonder "does the spec below select NVLink for me"? Can we look into this & recommend?